ui/src/components/Orchestration/MakeAccount.tsx

```diff
@@ -1,63 +1,13 @@
 import { AgoricWalletConnection, useAgoric } from '@agoric/react-components';
-import { SigningStargateClient, StargateClient } from '@cosmjs/stargate';
+import { StargateClient } from '@cosmjs/stargate';
 import { useContext, useEffect, useRef, useState } from 'react';
 import { Button } from 'react-daisyui';
 import { NotificationContext } from '../../context/NotificationContext';
 import { useContractStore } from '../../store/contract';
 import { DynamicToastChild } from '../Tabs';
+import { ibcChannels, rpcEndpoints } from '../../util';
 
-const rpcEndpoints = {
-  osmosis: 'http://127.0.0.1:26655',
-  agoric: 'http://127.0.0.1:26657',
-};
-
-const initializeKeplr = async () => {
```

Comment:
:clap: nice to see that dapps don't have to do this sort of thing.

Reply:
@Jovonni Hinted that this might be useful for local dev/debugging stuff (sending tokens around etc.), just by nature of having the network added to your keplr I assume. I guess we could put it back in and only invoke it when you select "Agoric Local" in the dropdown.

Reply:
yea it may come in handy in the near future, depending on what devs end up doing with the dapp. For example, when trying to set up a auto delegate scenario, you have to start with the ibc/denom on agoric-local, then submit it with the offer. 

i put a little convenience to help set it up, without having to use the dapp for now, so may not need on the UI:
https://github.com/Agoric/dapp-orchestration-basics/blob/249a6c2c08871024f31b778d166b282a2131fd4b/contract/Makefile#L162-L164